client/src/components/pages/profileBusiness/index.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react'
+import { Form, Input, AutoComplete, Cascader, Select } from 'antd'
+
+import Button from 'components/utils/Button'
+import { residences, BusinesTypeValues } from './static'
+import Options from "components/utils/option"
+import json from './country.json'
+
+const AutoCompleteOption = AutoComplete.Option;
+
+class BusinessForm extends React.Component {
+
+  state = {
+    autoCompleteResultWebsite: [],
+    autoCompleteResultCuntry: [],
+    autoCompleteResultCity: [],
+    country: ''
+  };
+
+  handleWebsiteChange = value => {
+    let autoCompleteResultWebsite;
+    if (!value) {
+      autoCompleteResultWebsite = [];
+    } else {
+      autoCompleteResultWebsite = ['.com', '.org', '.net'].map(domain => `${value}${domain}`);
+    }
+    this.setState({ autoCompleteResultWebsite });
+  };
+
+  handleCuntryChange = value => {
+    let autoCompleteResultCuntry;
+    if (!value) {
+      autoCompleteResultCuntry = [];
+    } else {
+      const countries = Object.keys(json)
+      let data = countries.filter((country) => {
+        return country.toLowerCase().startsWith(value.toLowerCase())
+      })
+      autoCompleteResultCuntry = data.map(domain => `${domain}`);
+    }
+    this.setState(() => {
+      return {
+        autoCompleteResultCuntry,
+        country: value
+      }
+    });
+  };
+
+  handleCityChange = value => {
+    const { country } = this.state
+    let autoCompleteResultCity;
+    if (!value) {
+      autoCompleteResultCity = [];
+    } else {
+      let data = json[country].filter((city) => {
+        return city.toLowerCase().includes(value.toLowerCase());
+      })
+      autoCompleteResultCity = data.map(domain => `${domain}`);
+    }
+    this.setState(() => {
+      return { autoCompleteResultCity }
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    this.props.form.validateFieldsAndScroll((err, values) => {
+      if (!err) {
+        console.log('Received values of form: ', values);
+      }
+    });
+  };
+
+  render() {
+    const { getFieldDecorator } = this.props.form;
+    const { autoCompleteResultWebsite, autoCompleteResultCuntry, autoCompleteResultCity } = this.state;
+
+    const formItemLayout = {
+      labelCol: {
+        xs: { span: 0 },
+        sm: { span: 0 },
+      },
+      wrapperCol: {
+        xs: { span: 0 },
+        sm: { span: 24 },
+      },
+    };
+    const tailFormItemLayout = {
```

Comment:
@AbdallahAmmar96 If formItemLayout and tailFormItemLayout are the same here as they are on other pages, it might be a good idea to pull them out into a static file and import them into all of the pages that need them.  Then you can easily change the form layout globally.